resources/plugins/plugins.json

```diff
@@ -1,8 +1,9 @@
-[    
-    "base",
-    "jtacautolase",
-    "skynetiads",
-    "ewrs",
-    "herculescargo",
-	"splashdamage"
-]
+[    
+    "base",
+    "jtacautolase",
+    "skynetiads",
+    "ewrs",
+    "herculescargo",
+  	"splashdamage",
```

Comment:
Seems this whole file was reformatted? Accidental line ending conversion?

This line in particular is indented wrong.

Reply:
Done